requirements/requirements.txt

```diff
@@ -18,7 +18,7 @@ django-kombu==0.9.1
 django-rosetta==0.7.1
 django-tastypie==0.9.14
 djtables==0.1.1
-egenix-mx-base==3.2.8
+#egenix-mx-base==3.2.8
```

Comment:
Yes. I've greped the repo in search of these modules but could'nt find any import :
* mxDateTime
* mxTextTools
* mxStack
* mxTools
* mxProxy
* mxBeeBase

`for i in mxDateTime mxTextTools mxStack mxTools mxProxy mxBeeBase ; do grep -Ri $i . ; done`